spec/rspec/rails/configuration_spec.rb

```diff
@@ -256,6 +256,31 @@ def self.application; end
       expect(group.mailer_class).to be(a_mailer_class)
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
+
+    describe 'cleans test mailbox between each example in all rspec-rails example' do
```

Comment:
Not only in `rspec-rails` example. If there's one test in a focused run that would require `rails_helper`, we'll be cleaning deliveries after each example, Rails or not.
Maybe just "leans test mailbox after each example"?

Reply:
https://github.com/rspec/rspec-rails/pull/2293/commits/f9d2dc8dd029077981cff6e21bde0efb1c343a03